### DISCUSSION TITLE
When do you plan to release GA ?

### DISCUSSION THREAD

**Discussion by gvart-sellics** [awspring/spring-cloud-aws#572] [Q&A] [answered]
Hey guys, you did a fantastic job, by moving to AWS SDKv2, providing preconfigured SQS/SNS/S3 clients, and application properties for easy setup of clients with localstack services, Advanced SQSListener setup with the backpressure support is insane, etc. 🎉 

I'd like to start using all the things you build in prod asap.

Do you have any estimation on GA release?

**Comment by gvart-sellics** [comment, marked as answer]
up

**Reply by maciejwalkowiak** [reply]
Apologies I missed your question. This is our scope for RC1: https://github.com/awspring/spring-cloud-aws/milestone/16. The biggest part is `SqsTemplate` that @tomazfernandes is working on. Considering the current progress I think it is reasonable to assume that RC1 will be released in January (ASAP) and we will make a buffer of 2-4 weeks to get feedback before we go GA.

**Reply by gvart-sellics** [reply]
tyvm, good luck with release

**Reply by solidjb** [reply]
@maciejwalkowiak - Thank you for your work on this project.  Do you have an updated GA date at this time?

**Reply by maciejwalkowiak** [reply]
No, considering number of changes we have since RC1 perhaps we will release RC2 first. Not sure yet.

**Reply by gauravc31** [reply]
Hi @maciejwalkowiak , We are waiting for the release to go GA to be able to use in our production, is there a timeline fixed yet for this?
It would be super helpful if we can get to a date for GA as many of us are waiting for the same.

**Reply by maciejwalkowiak** [reply]
Is there anything stopping you from using RC in production?

**Reply by PatrickGotthard** [reply]
We are also just waiting for an official release, it simply doesn't feel right to use a RC in production ;)

**Reply by maciejwalkowiak** [reply]
Ok ok I'll try to do it sooner than later then :)

**Reply by gauravc31** [reply]
@maciejwalkowiak Can you please help in giving a tentative date for the GA release?

**Reply by PatrickGotthard** [reply]
Today 😉

**Comment by CasaSky** [comment]
Hi @maciejwalkowiak can the RC1 be used in production or should we wait for the final one?

**Reply by maciejwalkowiak** [reply]
GA won't magically fix any bugs we have, so if you take into account that some APIs may slightly change, go for it. We don't plan any serious changes for GA, just filling holes.

**Reply by CasaSky** [reply]
i appreciate that! Tnx!

**Comment by arun1ram** [comment]
@maciejwalkowiak I am trying use Spring Boot 3. with Spring Cloud AWS 3.1.0. Based on https://github.com/awspring/spring-cloud-aws this should be compatible. But I am running into this issue: https://github.com/awspring/spring-cloud-aws/issues/1012
Any ideas?
